install/origin-web-console/console-template.yaml

```diff
@@ -48,8 +50,8 @@ objects:
         serviceAccountName: webconsole
         containers:
         - name: webconsole
-          image: ${IMAGE}
-          imagePullPolicy: IfNotPresent
+          image: ${OPENSHIFT_WEBCONSOLE_IMAGE}
```

Comment:
This is not going to scale well.

Reply:
why? btw. I moved these to the cluster add

Reply:
> why? btw. I moved these to the cluster add

What you did and what's needed is different. What did originally wont' scale.  in `cluster add` it ought to be ok.